### DISCUSSION TITLE
[sprint][210712] restructure scripts/build.d/ and launch to make them more friendly to be tested

### DISCUSSION THREAD

**Discussion by tai271828** [solvcon/devenv#47] [Ideas]
An open topic to plan.

**Comment by yungyuc** [comment]
Related issues: #44 #45 #48 #49

**Comment by yungyuc** [comment]
In addition to the issues resolved (#44 #45 #48 #49), one additional issue is open without resolution: #53.

This is a productive sprint!

**Reply by tai271828** [reply]
Productive +1

IIRC, @chestercheng will take over https://github.com/solvcon/devenv/issues/53 , right?

**Comment by tai271828** [comment]
My gut feeling shows the projects for the next sprint could be:
1. creating more test cases (as usual)
2. besides SOLVCON, building more applications
